frontend/src/lib/UriUtil.ts

```diff
@@ -115,3 +115,23 @@ export function buildMediaUri(uri: string): string {
     ? buildHttpUri(getWindowBaseUriParts(), uri)
     : uri
 }
+
+/**
+ * Check if the given url follows the url pattern which could include
+ * a wildcard.
+ */
+export function isValidURL(pattern: string, hostname: string): boolean {
+  const splittedPattern = pattern.split(".")
+  const splittedHostname = hostname.split(".")
+
+  if (pattern === hostname) return true
```

Comment:
nit: I would maybe move this to the first line of the function as we don't have to split if the two passed arguments are equal